iaac/terraform/aws-infra/rds/main.tf

```diff
@@ -0,0 +1,45 @@
+resource "aws_db_subnet_group" "rds_db_subnet_group" {
+  subnet_ids = var.subnet_ids
+}
+
+resource "random_uuid" "db_snapshot_suffix" {
+  keepers = {
+    instance_class       = var.db_class
+    db_name                 = var.db_name
+    username = var.db_username
+    password = var.db_password
+    multi_az = var.multi_az
+    db_subnet_group_name = aws_db_subnet_group.rds_db_subnet_group.id
+    security_group_id = var.security_group_id
+  }
+}
+
+resource "aws_db_instance" "kubeflow_db" {
+  allocated_storage    = var.db_allocated_storage
+  engine               = "mysql"
+  engine_version       = "8.0.30"
```

Comment:
can you please expose this as var as well?

Reply:
done